src/value/mod.rs

```diff
@@ -12,6 +12,7 @@
 //! [simd-json-value]: https://docs.rs/simd-json/latest/simd_json/value/index.html
 //! [serde-json-value]: https://docs.serde.rs/serde_json/value/enum.Value.html
 
+pub mod builders;
```

Comment:
🗺️ Changes in this file are superficial; I just updated the unit tests to use the new macros where possible.